bin/stop-coordinator.sh

```diff
@@ -21,6 +21,7 @@ set -o pipefail
 set -o nounset   # exit the script if you try to use an uninitialised variable
 set -o errexit   # exit the script if any statement returns a non-true return value
 
+UNIFFLE_SHELL_SCRIPT_DEBUG=true
```

Comment:
Could we put `UNIFFLE_SHELL_SCRIPT_DEBUG=true` into `utils.sh` and `load_rss_env`?

Reply:
I will modify the code.